.github/workflows/eo-cd.yml

```diff
@@ -165,19 +162,3 @@ jobs:
           configuration: build/infrastructure/eo/configuration.yaml
           deploy_key: ${{ secrets.DEPLOY_KEY_BASE_ENVIRONMENT }}
           dry_run: ${{ !(github.event_name == 'push' && github.ref_name == 'main' || github.event_name == 'pull_request' && startsWith(github.event.pull_request.head.ref, 'preview/')) }}
-
-  bump_configuration_version:
-    runs-on: ubuntu-latest
-    name: Bump configuration version
-    if: ${{ github.event_name == 'pull_request' && needs.affected.outputs.is-affected == 'true' }}
-    needs:
-      - affected
-      - update-environment
-    steps:
-      - uses: actions/checkout@v3
-
-      - uses: ./.github/actions/yaml-increment-version
```

Comment:
suggestion: as part of this PR, can you also delete the `.github/actions/yaml-increment-version` workflow? Its only purpose is to [automatically] bump the version, which will not be needed anymore.

praise: great job getting rid of the version bump 💯

Reply:
Can you verify that it's not being used anywhere else than the eo? We were not sure, but will gladly remove it, if its not being used anywhere.

Reply:
Yes, it's not used anywhere else. It can be removed.

Reply:
Awesome, thanks for the suggestion.